gslab_scons/tests/test_build_stata.py

```diff
@@ -47,32 +47,14 @@ def test_windows(self, mock_is_64, mock_check, mock_path):
         Test that build_stata() behaves correctly on a Windows machine
         when given appropriate inputs. 
         '''
-        # i) Non-64-bit Windows without a STATAEXE environment variable
-        mock_check.side_effect = fx.make_stata_side_effect('statamp.exe')
-        mock_path.side_effect  = fx.make_stata_path_effect('statamp.exe')
+        mock_check.side_effect = fx.make_stata_side_effect('StataMP-64.exe')
+        mock_path.side_effect  = fx.make_stata_path_effect('StataMP-64.exe')
         mock_is_64.return_value = False
 
         env = {'stata_executable' : None}
```

Comment:
may want to try this for the current set of values that trigger default options being `None, 'None', ''`